.prospector.yaml

```diff
@@ -7,6 +7,8 @@ test-warnings: false
 
 pep8:
   full: true
+  disable:
+    - N802
```

Comment:
@nishakm do you think it could be helpful to add a comment above or after this error code specifying what it is exactly?

Reply:
Let me try...

Reply:
@rnjudge I've added a comment